### PR TITLE
Migrate from gopkg.in to github.com, to support 'go mod' more cleanly

### LINKED ISSUE #228: Using go-jira as a dependency is broken with go modules ("vgo")
Trying to use `go-jira` in a `go.mod` file like this:

```
module github.com/optile/releasetool

require (
        github.com/Netflix-Skunkworks/go-jira v1.0.20
)
```

`go build` fails with

```
go: extracting gopkg.in/Netflix-Skunkworks/go-jira.v1 v1.0.20
-> unzip /home/jonas/go/pkg/mod/cache/download/gopkg.in/!netflix-!skunkworks/go-jira.v1/@v/v1.0.20.zip: malformed file path "t/.password-store/GoJira/api-token:gojira@corybennett.org.gpg": invalid char ':'
build github.com/optile/releasetool: cannot find module for path gopkg.in/Netflix-Skunkworks/go-jira.v1
```

Do we have a way to get rid of the offending file(s)?

### PR DESCRIPTION
**Pull request by mikepea**
I'm not 100% on this PR, so would really like a second pair of eyes from someone who's used to using modules.

This PR removes reference to the old repository and in particular the use of gopkg.in as an import source - it appears to have been preventing go module support from being useful for pinning against a non-released SHA or branch. Essentially, since gopkg.in only pulls vX.Y.Z tagged releases from github, there's nothing there to pull from if you want to use the 'go get {repo}@{git-sha} syntax.

In the process of this, I noticed that #228 was still a problem, despite the move to using '_t' as the prefix for the cli tests. I implemented a temp-file fix to address the problem, so it should now work - at least the tests pass.

### CODE REVIEW

**Comment by mikepea** [comment]
@coryb are you happy for me to merge this?

File: go.mod [hunk 1]
```diff
@@ -15,6 +15,7 @@ require (
 	github.com/guelfey/go.dbus v0.0.0-20131113121618-f6a3a2366cc3 // indirect
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c // indirect
 	github.com/jinzhu/copier v0.0.0-20180308034124-7e38e58719c3
+	github.com/karalabe/xgo v0.0.0-20190301120235-2d6d1848fb02 // indirect
```
Comment (coryb) [inline go.mod]: dont think we need this dep, it is only for our cross compile, not sure how to exclude it from the go.mod

**Comment by coryb** [comment]
I have only limited use of go modules, but it seems like to use non-released code you can use the `replace` rule? https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive
Maybe something like `replace gopkg.in/Netflix-Skunkworks/go-jira.v1 => github.com/go-jira/jira master` syntax in the go.mod?

**Comment by coryb** [comment]
I am not sure of the complications of moving away from gopkg.in, I like the idea of removing the layer of indirection,  but I think it will also break existing users using go modules given comments from this other PR #267.  I think the usage of this is very low, like just admins on this project, so I think we can make this change without to many complications.  Worst-case we can revert if we end up breaking people.

**Comment by mikepea** [comment]
I don't think we're going to affect folk using go modules -- I'm pretty sure modules support was broken until #228 was fixed, and to use modules effectively you'd need to do the replace trick to the current github repo anyway.

I disagree that we need to bump the major version in order to shift away from gopkg.in (re #267) - it's not really a breaking change, it just requires an update to your imports if you wish to use future versions - the code/API remains at v1.

I say let's do it :)

File: go.mod (hunk 1 above)
Comment (mikepea) [reply go.mod]: i've removed this as part of the clean-up rebase.

**Comment by mikepea** [comment]
I'm going to hold off on merging this until I've had a chance to check it with Go 1.13 too -- I've heard it surfaces some module problems that can take a bit of unpicking.

**Comment by coryb** [comment]
You can try to update the .travis.yml from:
```
go:
  - 1.12.x
```
to:
```
go:
  - 1.12.x
  - 1.13.x
```
That should let travis test for you

**Pull request merged**